### PR TITLE
Add CI for Stylelint and Prettier

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add CI check for Stylelint and Prettier
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | @PrestaShopCorp
| How to test?      | Check in "Actions" and "Web Integration Check" if the CI is green !

### CODE REVIEW

**Review by ga-devfront — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
